python/tests/test_account.py

```diff
@@ -2296,7 +2294,7 @@ def test_fetch_deleted_msg(self, acfactory, lp):
         ac1.direct_imap.delete("1:*", expunge=False)
         ac1.start_io()
 
-        for ev in ac1._evtracker.iter_events():
+        for ev in ac1._evtracker.iter_events(timeout=60):
```

Comment:
Can adding these timeouts be avoided? I removed them in #3059 to make it possible to run the tests on slow connections.